src/cli/cli.ts

```diff
@@ -161,7 +162,7 @@ export class Cli {
                 this.server.setOptions(settingObject);
             }
         } catch (e) {
-            //
+            Log.error('There was an error while parsing the JSON in your config file. It has not been loaded.');
```

Comment:
Can we make it more verbose-y? I think `Log.errorTitle()` would be better to be more visible? 🤔

Reply:
Sorry for the slow response - 

The styling of errorTitle is definitely an improvement although I'm not sure the timestamp is necessary given it's a one off error on load. Your call though. I'll commit the errorTitle change.

![soketi-errorTitle](https://user-images.githubusercontent.com/30434619/157024611-7466deef-bae2-4f81-b53a-aef61751ef04.png)